RELEASES.md

```diff
@@ -0,0 +1,16 @@
+## Versioning
+
+- Major.Minor.Patch
+- Major: updated if there is big changes in the architecture or APIs
```

Comment:
"there are big changes" or "there is big change(s?)", I am not sure, which one is right?

Reply:
Fixed